src/GitTools.Core/GitTools.Core.csproj

```diff
@@ -19,36 +19,36 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\output\debug\NET45\GitTools.Core\</OutputPath>
+    <OutputPath>..\..\output\Debug\GitTools.Core\net45\</OutputPath>
```

Comment:
I normally follow this structure:

[platform]\[product]

Since if you are building, you are build against a platform and are not interested in other stuff. But if this is what nuget requires...